src/main.go

```diff
@@ -655,20 +657,30 @@ OPTIONS:
 		cli.Command{
 			Name:        "new-boss",
 			Usage:       "Create an OL Boss config (boss.json)",
-			UsageText:   "ol new-boss [--path=PATH] [--detach]",
+			UsageText:   "ol new-boss [--path=PATH] [--detach] [--platform]",
```

Comment:
shouldn't platform be a setting in the .JSON file instead of a command line argument?

Take a look at how we use overrideOpts for the worker -- in this case, anything in the config can be modified with a "-o" option.  Maybe the same makes sense here?